frontend-nextjs/Dockerfile.gcloud-nextjs

```
@@ -0,0 +1,65 @@
+ENV NODE_ENV=production
+# Uncomment the following line in case you want to disable telemetry during runtime.
+ENV NEXT_TELEMETRY_DISABLED=1
+
+RUN addgroup --system --gid 1001 nodejs
+RUN adduser --system --uid 1001 nextjs
+
+COPY --from=builder /app/public ./public
+
+# Set the correct permission for prerender cache
+RUN mkdir .next
+RUN chown nextjs:nodejs .next
```

Hadolint

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)